src/main/webapp/i18n/de/metis.json

```diff
@@ -37,6 +37,9 @@
             "reactedTooltipTrimmed": " und {{ number }} weitere Person(en) haben reagiert.",
             "searchEmoji": "Emoji suchen",
             "courseEmojiSelectionCategory": "Ausgewählte Emojis für diesen Kurs",
+            "editor": {
+                "exercise": "Übung"
```

Comment:
We translated `Exercise` as `Aufgabe` in `assessment.json` - i guess we should stay consistent here :thinking: